lib/cocoapods/validator.rb

```diff
@@ -821,7 +821,8 @@ def translate_output_to_linter_messages(parsed_output)
     end
 
     def shares_pod_target_xcscheme?(pod_target)
-      Pathname.new(@installer.pods_project.path + pod_target.label).exist?
+      scheme_path = Xcodeproj::XCScheme.user_data_dir(@installer.pods_project.path) + "#{pod_target.label}.xcscheme"
```

Comment:
I still don't understand the fix here. Can you give a path example of what works today and one that doesn't?